app/models/assignment.js

```diff
@@ -84,12 +75,57 @@ export default class AssignmentModel extends Model {
     return this.project.get('milestones').filter(milestone => this.milestoneConstants.milestoneListByTabLookup[this.tab].includes(milestone.dcpMilestone));
   }
 
+  @computed('project.milestones')
+  get assigneeMilestoneIdentifier() {
+    return this.milestoneConstants.referralIdentifierByAcronymLookup[this.dcpLupteammemberrole];
+  }
+
+  @computed('project.milestones')
+  get lastCompletedMilestone() {
+    const completedMilestones = this.project.get('milestones').filterBy('statuscode', 'Completed');
+
+    return completedMilestones[completedMilestones.length - 1] || null;
+  }
+
+  // abridged view of milestones typically used in upcoming tab
+  @computed('tab', 'project.milestones')
```

Comment:
Could you add a comment saying what's supposed to show up on the upcoming tab? That way if someone goes in to change the logic they know exactly what's supposed to show up. e.g. "the upcoming tab should show 1) milestones that blah blah, 2) milestones that blah blah etc."